src/main/java/org/jabref/gui/exporter/ExportAction.java

```diff
@@ -33,6 +33,9 @@
 
     private static final Log LOGGER = LogFactory.getLog(ExportAction.class);
 
+    private ExportAction() {
```

Comment:
I just had a look at that. `ExportAction` is indeed instantiated in `org.jabref.gui.preftabs.PreferencesDialog`, but that class has its own `ExportAction` class in it. `PreferencesDialog` also has no import for `org.jabref.gui.exporter.ExportAction`, so the instantiation is most likely an instantiation of the inner class.